### ISSUE TITLE
BrainJS VS TensorFlow 

### ISSUE THREAD

**Issue by saleh-old** [BrainJS/brain.js#263]
Hey guys. 

I'm a pretty good JS developer and I favor JS compared to Python (for some reasons). Which is why I decided to use TenserFlowJS (now that it's out). I liked it until I found this library which is like crazy simple and seems to meet my use case. However accuracy (as in how good it can do predictions) is very important to my use case. 

So my question is: How accurate is BrainJS compared to TensorFlowJS? I searched and couldn't find anything on this topic, probably because TensorFlowJS is pretty new. I'd like to hear some thoughts of you guys.

**Comment by robertleeplummerjr** [comment]
Accuracy in both would be equivalent.  A trained net is a trained net.  In tensorflow.js the training algorithms may allow the net to train a bit faster or slower, I believe there are several available for tensorflow where as we only have one available in v1, which is just standard gradient descent.

Brain.js v2 has Adam with momentum under the hood, so look forward to that.

**Comment by saleh-old** [comment]
Will do. Thanks for the response.
